src/pages/TelaCadastroInfoPeossoais.js

```diff
@@ -0,0 +1,166 @@
+import {
+  Text,
+  SafeAreaView,
+  StyleSheet,
+  TextInput,
+  View,
+  TouchableHighlight,
+} from 'react-native';
+import { useFonts } from 'expo-font';
+import AppLoading from "./AppLoading"
+import { useNavigation } from '@react-navigation/native';
+import routes from '../routes';
+import { useForm, Controller } from 'react-hook-form';
+
+// Step 01
+export default function TelaCadastroInfoPessoais() {
+  const navigate = useNavigation().navigate
+  const { control, handleSubmit, formState: { errors } } = useForm();
+
+  let [fontsLoaded] = useFonts({
+    'Barlow-Light': require("../assets/fonts/Barlow-Light.ttf"),
+  });
+  if (!fontsLoaded) {
+    return <AppLoading />
+  }
+
+  function onSubmit() {
+    // todo update user data from context
+    navigate(routes.TELA_CADASTRO_ENDERECO)
+  }
+
+  function handleKeyDown(e) {
+    if (e.nativeEvent.key == "Enter") {
+        Keyboard.dismiss();
+    }
+  }
+
+  return (
+    <SafeAreaView>
+      <View style={styles.div}>
+        <View style={styles.header}>
+          <Text style={styles.dogtorText}>DOGTOR</Text>
+          <Text style={{ fontWeight: 'bold', fontSize: 24 }}>Iformações pessoais</Text>
```

Comment:
Styles fora da variável styles

```suggestion
          <Text style={styles.title}>Iformações pessoais</Text>
```